<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QDialog" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>691</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_now_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>289</y>
     <width>125</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_3">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>289</y>
     <width>125</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>289</y>
     <width>125</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_3">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>289</y>
     <width>125</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>289</y>
     <width>125</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_3">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>289</y>
     <width>125</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_now_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>225</y>
     <width>125</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_2">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>225</y>
     <width>125</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>225</y>
     <width>125</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_2">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>225</y>
     <width>125</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>225</y>
     <width>125</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_2">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>225</y>
     <width>125</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6_1">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>125</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_now_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>125</y>
     <width>125</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2_1">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>125</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>125</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4_1">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>125</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5_1">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>125</y>
     <width>125</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
